packages/loopring_v3/contracts/amm/libamm/AmmExitProcess.sol

```diff
@@ -205,6 +205,7 @@ library AmmExitProcess
         uint newTotalSupply = ctx.totalSupply.sub(exit.burnAmount);
         for (uint i = 0; i < ctx.tokens.length; i++) {
             ctx.vTokenBalancesL2[i] = (uint(ctx.vTokenBalancesL2[i]).mul(newTotalSupply) / ctx.totalSupply).toUint96();
+            require(ctx.vTokenBalancesL2[i] > 0, "ZERO_VIRTUAL_BALANCE");
```

Comment:
If the pool is empty after the exit, pool supply will be `0` so the virtual balances will also be `0`.

Reply:
I see